homeassistant/components/mysensors.py

```diff
@@ -428,3 +596,29 @@ def update(self):
                 self._values[value_type] = int(value)
             else:
                 self._values[value_type] = value
+
+
+class MySensorsEntity(MySensorsDevice, Entity):
+    """Representation of a MySensors entity."""
+
+    @property
+    def should_poll(self):
+        """Mysensor gateway pushes its state to HA."""
+        return False
+
+    @property
+    def available(self):
+        """Return true if entity is available."""
+        return self.value_type in self._values
+
+    def _update_callback(self):
```

Comment:
Make this async and call `self.hass.async_add_job(self.async_update_ha_state(True))`